Client/src/scss/style.scss

```diff
@@ -128,6 +128,24 @@
   border: darkblue 1px solid;
 }
 
+.buy-now-span p {
+  white-space: nowrap;
+  border-radius: 25px;
+  display: block;
+  float: center;
+  text-align: center;
+  color: white;
+  backGround-color: blue;

```

Comment:
it may work as-is, but I don't think the G in background is supposed to be capitalized